examples/Suite2p example.ipynb

```diff
@@ -135,8 +135,8 @@
    "outputs": [],
    "source": [
     "output_folder = \"fissa_suite2p_example\"\n",
-    "exp = fissa.Experiment(images, [rois[:ncells]], output_folder)\n",
-    "exp.separate()"
+    "experiment = fissa.Experiment(images, [rois[:ncells]], output_folder)\n",
+    "experiment.separate()"
```

Comment:
Sorry, I still don't have an environment that runs suite2p, so I rewrote these plots by running them in another notebook and copying the code over to this one.